### PR TITLE
chore: address .NET 8 linter warnings

### PR DESCRIPTION
**Pull request by mxschmitt**
https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0016?view=aspnetcore-8.0
https://learn.microsoft.com/en-us/aspnet/core/diagnostics/asp0019?view=aspnetcore-8.0

### CODE REVIEW

**Review by yury-s — APPROVED** [review]

File: src/Playwright.MSTest/ContextTest.cs [hunk 1]
```diff
@@ -40,6 +40,10 @@ public async Task ContextSetup()
 
     public virtual BrowserNewContextOptions ContextOptions()
     {
-        return null!;
+        return new()
+        {
+            Locale = "en-US",
```
Comment (yury-s) [inline src/Playwright.MSTest/ContextTest.cs:45]: This seems irrelevant to .net 8

**Review by yury-s — APPROVED** [review]

File: src/Playwright.MSTest/ContextTest.cs (hunk 1 above)
Comment (mxschmitt) [reply src/Playwright.MSTest/ContextTest.cs:45]: Discussed offline: Align it with the upstream defaults.
